server/config/fbSessions.js

```diff
@@ -2,7 +2,13 @@ var passport = require('passport');
 var cookieParser = require('cookie-parser');
 var session = require('express-session');
 var FacebookStrategy = require('passport-facebook').Strategy;
-var api_keys = require('./api_keys.js').facebook;
+
+var api_keys;
+
+// load apikeys if localhost. process.env.DEPLOYED set in heroku
+if (!process.env.DEPLOYED) {
+  api_keys = require('../config/api_keys.js').facebook;

```

Comment:
oh wow.. I have no idea why I did that..... it means I rewrote that line... why....